skymp5-server/cpp/server_guest_lib/script_classes/PapyrusUtility.cpp

```diff
@@ -48,6 +48,15 @@ VarValue PapyrusUtility::RandomFloat(VarValue self,
   return VarValue(distribute(g_generator));
 }
 
+VarValue PapyrusUtility::GetCurrentRealTime(
+  VarValue self, const std::vector<VarValue>& arguments)
+{
+  return VarValue(std::chrono::duration<double, std::micro>(
```

Comment:
https://ck.uesp.net/wiki/GetCurrentRealTime_-_Utility

please make sure this is seconds (standard behavior), not microseconds

Reply:
Rechecked and changed.